### DISCUSSION TITLE
Prev month button

### DISCUSSION THREAD

**Discussion by andykoe24** [nhn/tui.calendar#1267] [Q&A] [answered]
Hi,
i have a question
is it possible to get the month name that it go to?
and alsois there a way to get current display month name?

i am using v2.1.3

**Comment by Tragio** [comment, marked as answer]
@andykoe24 

You can use `getDate` https://nhn.github.io/tui.calendar/latest/CalendarCore#getDate to get the current rendered date. With that date, you can now parse it using JS. https://stackoverflow.com/questions/1643320/get-month-name-from-date/18648314#18648314
